tests/parsers/fixtures/pw2gw/default/epsTOT.dat

```diff
@@ -0,0 +1,100 @@
+       0.000500        0.000000
```

Comment:
Sorry to come back to this, but I think they can be reduced to a single line. Your parser is really just calling `numpy.loadtxt` and checking that all output files have the shape `(N, 2)`. To test this, just a single line suffices, no?

Reply:
I will reduce it to 2 lines in order t check the possibility of
```suggestion
       0.000500
       0.001000        0.000000
```